draft-ietf-dnsop-svcb-https.md

```diff
@@ -1212,12 +1212,14 @@ Consider a simple zone of the form:
 
 The domain owner could add this record:
 
-    simple.example. 7200 IN HTTPS 1 . alpn=h3
+    @ 7200 IN HTTPS 1 . alpn=h3
 
-to indicate that simple.example uses HTTPS, and supports QUIC
+to indicate that https://simple.example uses HTTPS, and supports QUIC
 in addition to HTTPS over TCP (an implicit default).
-The record could also include other information
-(e.g. non-standard port, ECH configuration).
+The record could also include other information (e.g. non-standard port,
+ECH configuration).  For https://simple.example:8443, the record would be:
+
+    _8443._https 7200 IN HTTPS 1 . alpn=h3
 
```

Comment:
Corner-case that I'm not sure we cover.  What happens to http://simple.example:8443 in this case?  Does it also get forced to HTTPS?

Reply:
Yes, it does.  I added this to the last sentence of this example.